doc/man7/EVP_MD-MD5-SHA1.pod

```diff
@@ -0,0 +1,63 @@
+=pod
+
+=head1 NAME
+
+EVP_MD-MD5-SHA1 - The MD5-SHA1 EVP_MD implementation
+
+=head1 DESCRIPTION
+
+Support for computing MD5-SHA1 digests through the B<EVP_MD> API.
+
+MD5-SHA1 is a rather special digest that's used with SSL v3.
+
+=head2 Identity
+
+This implementation is identified with this name and properties, to be
+used with EVP_MD_fetch():
+
+=over 4
+
+=item "MD5-SHA1", "provider=default"
+
+=back
+
+=head2 Gettable Parameters
+
+This implementation supports the common gettable parameters described
+in L<EVP_MD-common(7)>.
+
+=head2 Settable Context Parameters
+
+This implementation supports the following L<OSSL_PARAM(3)> entries,
+settable for an B<EVP_MD_CTX> with L<EVP_MD_CTX_set_params(3)>:
+
+=over 4
+
+=item "ssl3-ms" (B<OSSL_DIGEST_PARAM_SSL3_MS>) <octet string>
+
+This parameter is set by libssl in order to calculate a signature hash for an
+SSLv3 CertificateVerify message as per RFC6101.
+It is only set after all handshake messages have already been digested via
+OP_digest_update() calls.
+The parameter provides the master secret value to be added to the digest.
+The digest implementation should calculate the complete digest as per RFC6101
+section 5.6.8.
+The next call after setting this parameter will be OP_digest_final().
+This is only relevant for implementations of SHA1 or MD5_SHA1.
```

Comment:
Does this final sentence need to be said since we're in the MD5-SHA1 description?

Reply:
Ah, no. I copied and pasted, but didn't look more closely. Will fix